olcPGEX_AdditionalColours.h

```diff
@@ -6,13 +6,81 @@ namespace olc
 {
 	namespace Colours
 	{
+		//ALL THE COLORS OF THE WIND!
 		static const Pixel
+			BLACK(24, 24, 26),
+			WHITE(239, 248, 254),
+
+			GREY(211, 231, 232),
+			DARK_GREY(123, 135, 143),
+			VERY_DARK_GREY(58, 61, 69),
+
+			RED(230, 65, 48),
+			DARK_RED(193, 14, 0),
+			VERY_DARK_RED(77, 18, 13),
+
 			ORANGE(255, 140, 0),
-			DARK_ORANGE(255, 127, 0),
-			VERY_DARK_ORANGE(255, 114, 0),
+			DARK_ORANGE(176, 100, 0),
```

Comment:
Do these still look orange? I know when Moros was trying it, the darker oranges ended up looking more like brown.